context/app/static/js/components/Header/UserLinks/UserLinks.spec.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-unresolved */
+import React from 'react';
+import { render, screen } from 'test-utils/functions';
+import userEvent from '@testing-library/user-event';
```

Comment:
Since we started writing tests `@testing-library/react` has moved towards `fireEvent`. https://testing-library.com/docs/react-testing-library/example-intro. Not a huge deal.